samples/nrf9160/modem_shell/README.rst

```diff
@@ -1135,6 +1135,37 @@ For example:
 
    west build -p -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG=overlay-rtt.conf
 
+LwM2M support
+=============
+
+To build the MoSh sample with LwM2M support, use the ``-DOVERLAY_CONFIG=overlay-lwm2m_client.conf`` option and set the used Pre-Shared-Key (PSK) using :kconfig:option:`CONFIG_MOSH_LWM2M_PSK`.
+The LwM2M server address can be changed by setting :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_SERVER`.
+P-GPS support can be enabled by using the optional overlay ``overlay-lwm2m_client_pgps.conf``.
+
+For example:
+
+.. code-block:: console
+
+   west build -p -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG=overlay-lwm2m_client.conf -DCONFIG_MOSH_LWM2M_PSK=\"000102030405060708090a0b0c0d0e0f\"
```

Comment:
Maybe worth to add also example of building with p-gps support

Reply:
Yes, will do.